stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -135,6 +136,10 @@ spec:
           failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
         {{- end }}
         volumeMounts:
+        {{- if .Values.usePasswordFile }}
+        - name: postgresql-password
+          mountPath: /opt/bitnami/postgresql/secrets
+        {{ end }}
```

Comment:
Should we mount it as `readOnly`?

```
{{- if .Values.usePasswordFile }}
- name: postgresql-password
  mountPath: /opt/bitnami/postgresql/secrets
  readOnly: true
{{ end }}
```

Reply:
This is done by default when we are using secrets and configmaps

Reply:
👍